doc/dev/api/authentication.rst

```diff
@@ -0,0 +1,119 @@
+.. _`api_authentication`:
+
+
+Authentication
+==============
+
+In order to use the API you need to authenticate yourself. In future there
+will be multiple ways to do use, currently the only option is via an token.
+
+Token Authentication
+--------------------
+
```

Comment:
We should add a description which type of tokens and which length is returned around here.

Reply:
See 582b539